Registry.php

```diff
@@ -113,6 +106,23 @@ public static function set ( $index, $value ) {
         return;
     }
 
+    /**
+     * Set a new multiple register
+     * @access public
+     * @param array $contain
+     * @return void
+     */
+    public static function store (array $contain) {
+
+        if(false === is_array($contain))
```

Comment:
Not needed as it's already defined as type of argument.

Reply:
Ok thanks